Resources/ServerInfo/Guidebook/DeltaV/Rules/CommunityRules/E2_Community.xml

```diff
@@ -6,6 +6,6 @@
   - Be respectful towards other members and avoid making others uncomfortable.
   - We have a zero tolerance policy for racism, sexism, homophobia, violence, threats, hate speech, slurs, and other forms of bigotry, discrimination, and harassment. 
     Slurs and terms that use real or implied mental or physical disability to mock or denigrate members or their in-game characters are also strictly prohibited. Failure to comply with this rule will result in community removal.
-  - The denigration of other player characters through reference to their species/morphotype or debasement by comparison to nonsophonts on a discriminatory basis is strictly prohibited and may result in an appeal-only ban.
+  - The denigration of other player characters through reference to their species/morphotype, their status as a Borg, or any debasement by comparison to nonsophonts on a discriminatory basis is strictly prohibited and may result in an appeal-only ban.
```

Comment:
```suggestion
  - The denigration of other player characters through reference to their species/morphotype, their status as a cyborg, or any debasement by comparison to nonsophonts on a discriminatory basis is strictly prohibited and may result in an appeal-only ban.
```
Ignoring that this is too verbose and incomprehensible to anyone without a steadfast grasp on English vernacular; appropriate phrasing would refer to Cyborgs rather than a colloquial "Borg"

Reply:
Thanks for the feedback, I cleared it up a bit